common/scripts/update_bbs_accountIntegrations.sh

```diff
@@ -0,0 +1,65 @@
+#!/bin/bash -e
+
+readonly BBS_MIGRATION_VERSION=v6.3.4
+SHOULD_RUN_BBS_MIGRATION=false
+export COMPONENT="db"
+export DB_DATA_DIR="$RUNTIME_DIR/$COMPONENT/data"
+export DB_CONFIG_DIR="$CONFIG_DIR/$COMPONENT"
+export LOGS_FILE="$RUNTIME_DIR/logs/$COMPONENT.log"
+
+## Write logs of this script to component specific file
+exec &> >(tee -a "$LOGS_FILE")
+
+__compare_versions() {
+  local current_release="$1"
+  if [ "$current_release" != "master" ]; then
+    if [  "$current_release" = "`echo -e "$current_release\n$BBS_MIGRATION_VERSION" | sort -V | head -n1`" ]; then
```

Comment:
`=`?

Reply:
updating, it works with this too though.